### DISCUSSION TITLE
[Feature] (mobile) iOS keep album assignment

### DISCUSSION THREAD

**Discussion by JohannesHoffmann** [immich-app/immich#1845] [Feature Request]
### The feature

Inside the iOS App (do not have android and idk the workflow for backups there) you have to select the albums to backup. Unfortunately the images uploaded to the server loses their assignment of the album because no album was created in the immich album section.

**Good behaviour:**
Syncing / backuping albums leads to album generation in immich and adding the uploaded images to the album so that you have the same albums in immich that you selected from the app.

I couldn't find any settings to enable this feature neither related issues.

Version of Immich Server

1.48.1

Version of Immich Mobile App

1.48.0 build.87

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by My-ThrowAwayAccount** [comment]
Possible duplicate of #1678. Though I completely agree and hope this feature will be implemented, honestly that seems to be the only thing holding me back from switching.
